imageprocessor/processorcommand/gm.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/Imgur/mandible/imageprocessor/thumbType"
 )
 
-const GM_COMMAND = "convert"
+const GM_COMMAND = "gm convert"
```

Comment:
Actually the `runProcessorCommand` function is used more generically for other types of commands then gm. Perhaps a new function that wraps `runProcessorCommand`?